cmd/ping/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/getlantern/ping"
+)
+
+func main() {
+	opts := &ping.Opts{}
+	flag.IntVar(&opts.Count, "c", ping.DefaultCount, "number of times to ping")
+	flag.IntVar(&opts.PayloadSize, "s", ping.DefaultPayloadSize, "number of bytes payload in each ICMP echo request")
+	flag.Parse()
+
+	if len(flag.Args()) == 0 {
+		log.Fatal("Please specify a host")
+	}
+
+	host := flag.Args()[0]
+	stats, err := ping.Run(host, opts)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Printf("\"%v\",%v,%v,%v,%v,%v\n", host, stats.RTTMin, stats.RTTAvg, stats.RTTMax, stats.RTTDev, stats.PLR)
```

Comment:
stats.RTTDev doesn't exist.  Old code?

Reply:
Also it'd be nice to have a header or something for these values.  I'm guessing the current format is to make this machine-parsable.  Maybe there could be a human-readable flag?  Could just be left for future work though since there's not really a use case for it now.

Reply:
Oh yeah thanks, I had to remove that because it's not supported on Windows. Fixed.